Resources/Private/BackendModule/src/Model/ResultCollection.ts

```diff
@@ -0,0 +1,6 @@
+import {ModuleItem} from "./ModuleItem";
+
+export interface ResultCollection {
```

Comment:
Ist eigentlich kein Result, und auch keine Collection.

In sich logisch wäre denke ich das hier, inspiriert von Laravel:
```
export interface PaginagedCollection {
    items: ModuleItem[],
    currentPage: number
    perPage: number
}
```